pkg/fanal/artifact/image/image.go

```diff
@@ -36,6 +40,8 @@ type Artifact struct {
 	handlerManager handler.Manager
 
 	artifactOption artifact.Option
+
+	cacheDir string
```

Comment:
nit: Trivy already has several kinds of cache, like scan cache, cache assets, etc. `cacheDir` looks like the root directory of the cache directory controlled by `--cache-dir`. I'd rename it to the explicit name.

```suggestion
	layerCacheDir string
```

Reply:
Done https://github.com/aquasecurity/trivy/pull/8178/commits/bba405b6f2efce6e051aa02e0bb8d477961a1307